tracer/build/PackageVersionsLatestMajors.g.props

```diff
@@ -13,45 +13,35 @@ NOTE:  This code was generated by the GeneratePackageVersions tool. To safely
-      <Properties>ApiVersion=3.7.100.62;RestoreRecursive=false;BuildProjectReferences=false</Properties>
+      <Properties>ApiVersion=3.7.100.79;RestoreRecursive=false;BuildProjectReferences=false</Properties>
       <TargetFramework>netcoreapp2.1</TargetFramework>
       <RequiresDockerDependency>True</RequiresDockerDependency>
     </PackageVersionSample>
     <PackageVersionSample Include="test\test-applications\integrations\Samples.AWS.SQS\Samples.AWS.SQS.csproj">
-      <Properties>ApiVersion=3.7.100.62;RestoreRecursive=false;BuildProjectReferences=false</Properties>
+      <Properties>ApiVersion=3.7.100.79;RestoreRecursive=false;BuildProjectReferences=false</Properties>
       <TargetFramework>netcoreapp3.0</TargetFramework>
       <RequiresDockerDependency>True</RequiresDockerDependency>
     </PackageVersionSample>
     <PackageVersionSample Include="test\test-applications\integrations\Samples.AWS.SQS\Samples.AWS.SQS.csproj">
-      <Properties>ApiVersion=3.7.100.62;RestoreRecursive=false;BuildProjectReferences=false</Properties>
+      <Properties>ApiVersion=3.7.100.79;RestoreRecursive=false;BuildProjectReferences=false</Properties>
       <TargetFramework>netcoreapp3.1</TargetFramework>
       <RequiresDockerDependency>True</RequiresDockerDependency>
     </PackageVersionSample>
     <PackageVersionSample Include="test\test-applications\integrations\Samples.AWS.SQS\Samples.AWS.SQS.csproj">
-      <Properties>ApiVersion=3.7.100.62;RestoreRecursive=false;BuildProjectReferences=false</Properties>
+      <Properties>ApiVersion=3.7.100.79;RestoreRecursive=false;BuildProjectReferences=false</Properties>
       <TargetFramework>net5.0</TargetFramework>
       <RequiresDockerDependency>True</RequiresDockerDependency>
     </PackageVersionSample>
     <PackageVersionSample Include="test\test-applications\integrations\Samples.AWS.SQS\Samples.AWS.SQS.csproj">
-      <Properties>ApiVersion=3.7.100.62;RestoreRecursive=false;BuildProjectReferences=false</Properties>
+      <Properties>ApiVersion=3.7.100.79;RestoreRecursive=false;BuildProjectReferences=false</Properties>
       <TargetFramework>net6.0</TargetFramework>
       <RequiresDockerDependency>True</RequiresDockerDependency>
     </PackageVersionSample>
     <PackageVersionSample Include="test\test-applications\integrations\Samples.AWS.SQS\Samples.AWS.SQS.csproj">
-      <Properties>ApiVersion=3.7.100.62;RestoreRecursive=false;BuildProjectReferences=false</Properties>
+      <Properties>ApiVersion=3.7.100.79;RestoreRecursive=false;BuildProjectReferences=false</Properties>
       <TargetFramework>net7.0</TargetFramework>
       <RequiresDockerDependency>True</RequiresDockerDependency>
     </PackageVersionSample>
-    <PackageVersionSample Include="test\test-applications\integrations\Samples.MongoDB\Samples.MongoDB.csproj">
-      <Properties>ApiVersion=2.19.0;RestoreRecursive=false;BuildProjectReferences=false</Properties>
```

Comment:
it's weird to drop some entries on that side, I would have expected to see only additions of GraphQL ones

Reply:
The target framework is .NET461 and it is not available for macos (and so, not installed on my M1 machine). So it removed it.